### ISSUE TITLE
Wheat Seeds

### ISSUE THREAD

**Issue by JordenOnTheHub** [Marwinkas/MysticalCrops#11]
You can't put Wheat Seeds in the ritual table to craft Air Seeds

**Comment by Mudkipstalker** [comment]
Sooo i figured out the issue and fixed it myself, it has to do with the slot it needs having the minecraft:seeds tag being disabled when it's need. Funny enough it's also called "NoSeedSlot" which doesn't make sense when it literally adds a recipe in the game that NEEDS a seed from base minecraft.
![image](https://user-images.githubusercontent.com/26930675/201279796-a1808ff9-0466-4add-83c3-3ca0b3b87b92.png)
Basically setting this to true fixes the issue entirely.
